src/html/sections/intro.html

```diff
@@ -14,27 +14,23 @@
 		</picture>
 		<picture>
 			<source srcset="img/intro/background/x1920/bushLeft_x1920.png" type="image/jpeg"
-				media="(min-width:1920px)">
+				media="(min-width:1920px)" width="212px" height="435px">
```

Comment:
please, remove all the attributes with width and height and use css for it

Reply:
Done